tests/python/rest_api/test_tasks.py

```diff
@@ -15,9 +18,17 @@
 
 from shared.utils.config import get_method, make_api_client, patch_method
 from shared.utils.helpers import generate_image_files
+from shared.utils.s3 import make_client
```

Comment:
I suggest to import this s3 helper as a package, this will make calls like `make_client()` much more clear: `s3.make_client()`. We already have several SDK and test helpers called `make_client`, it's simple to confuse them.

Reply:
Fixed